src/plays/randombackgroundgenerator/Readme.md

```diff
@@ -0,0 +1,10 @@
+# Random Background Generator
```

Comment:
Add more details about the play

- What is it about
- How you created
- What libraries used
- What challenges faced

Consider someone will read it and learn

Reply:
Hi @Dev-Shabbir 

Could you please update the PR as I asked?

Reply:
Sir, I was in a trouble. I'm not so good in version control but learning faster. Can you please give a check.